nodewatcher/nodewatcher.py

```diff
@@ -203,12 +170,20 @@ def main():
         format='%(asctime)s %(levelname)s [%(module)s:%(funcName)s] %(message)s'
     )
     log.info('nodewatcher startup')
+    region, scheduler, stack_name, scaledown_idletime, proxy_config = _get_config()
+
     instance_id = _get_metadata("instance-id")
     hostname = _get_metadata("local-hostname")
-    log.info('Instance id is %s, hostname is %s' % (instance_id, hostname))
-    region, asg_name, scheduler, proxy_config, idle_time, stack_name = _get_config(instance_id)
+    log.info('Instance id is %s, hostname is %s', instance_id, hostname)
+
+    # get ASG name
```

Comment:
It will be do in a following patch.
We already have the get_asg_name function in the jobwatcher code, I'm going to move it to the common/utils file and use in both the files.

Reply:
Done in https://github.com/aws/aws-parallelcluster-node/pull/108.